### DISCUSSION TITLE
Command "sp:install" is not defined

### DISCUSSION THREAD

**Discussion by angemort** [mettle/sendportal#132] [Help]
Hello, after following the installation procedure of SendPortail.
I am stuck at the command "**php artisan sp:install**", it seems that the command does not exist.

Did you change your artisan command?

Or does the error come from my installation? I have tried the installation twice.

Thanks in advance for your help.

EDIT: Solution go through the installation without package.

**Comment by blake-ascent** [comment]
@angemort so you were unable to get SendPortal to work as a package for your laravel application? 
I ask because I am running into the same issue after having followed the instructions to add SendPortal as a package in my laravel application. Is it not supported?

**Comment by erhanask** [comment]
Any updates on this ?
